oshc/main/regbackend.py

```
@@ -0,0 +1,21 @@
+from django.contrib.auth import get_user_model
+
+
+class EmailBackend(object):
+    def authenticate(self, username=None, password=None):
+        user_cls = get_user_model()
+        try:
+            user = user_cls.objects.get(email=username)
+            if user.check_password(password):
+                return user
+        except user_cls.DoesNotExist:
+            return None
+        except:
```

You should never use a bare except clause. Read [this](https://stackoverflow.com/questions/18982610/difference-between-except-and-except-exception-as-e-in-python) for more info.

Both the except clause perform the same action, you should consider using a single except clause instead of two.
A single `except user_cls.DoesNotExists` should be sufficient.